data/json/furniture.json

```diff
@@ -222,6 +233,7 @@
         "name": "toilet",
         "symbol": "&",
         "color": "white",
+        "description": "Emergency water source, and provider of relief.",
```

Comment:
Would be nice if it mentioned that the water comes from the tank above. It's a common source of confusion.

Reply:
If the water comes from the tank above why does it make you sick then? Technically it should be clean.

Reply:
All water is unclean in the sense that it contains goo (it's how stuff was Z-ified in the first place)